server/index.js

```diff
@@ -2,43 +2,101 @@
 /* eslint-disable no-unused-vars */
 /* eslint-disable no-console */
 const express = require('express');
+
+const app = express();
 const axios = require('axios');
 const utils = require('../database/utils.js');
 const db = require('../database/index.js');
 
-const app = express();
 const port = 3004;
 
 app.use('/items/:itemId', express.static('client/dist'));
 app.use(express.json());
-app.use(express.urlencoded());
+app.use(express.urlencoded({ extended: false }));
 
-// Get all item data from db
+// Get all items from db
 app.get('/shopping/items', (req, res) => {
-  console.log('GET request for all items successful');
-  db
-    .query('SELECT * FROM items')
-    .then((result) => {
-      console.log(result.rows);
-      res.send(result.rows);
-    })
-    .catch((error) => {
-      console.error(error);
-      res.send('Error getting data from db');
-    });
+  db.model.getAll((err, data) => {
+    if (err) {
+      console.log('failed to get all data:', err);
+      res.status(400).send(err);
+    } else {
+      console.log('successfully retrieved all data');
+      res.status(200).send(data.rows);
```

Comment:
Hey, FYI I believe the send in a success case will automatically send 200. You can double check, but this may be redundant and save you a few key strokes.

Reply:
didn't know this, thanks!